packages/runtime/test-runtime-utils/src/mocks.ts

```diff
@@ -902,6 +906,9 @@ export class MockFluidDataStoreRuntime
 	}
 
 	public async resolveHandle(request: IRequest): Promise<IResponse> {
+		if (request.url !== undefined) {
+			return request.url as any as IResponse;
```

Comment:
you actually need to return an IResponse object here. This is probably why the tests are failing